int32 set_zv
---
int32 get_zv